addons/service.xbmc.versioncheck/lib/common.py

```diff
@@ -76,7 +76,7 @@ def get_password_from_user():
     return pwd
 
 def message_upgrade_success():
-    xbmc.executebuiltin("XBMC.Notification(%s, %s, %d, %s)" %(ADDONNAME,
+    xbmc.executebuiltin("Notification(%s, %s, %d, %s)" %(ADDONNAME,
```

Comment:
Isn't this being submitted to the repo atm? We need to make sure it doesn't get unnoticed in the PR

Reply:
no worries, it's no longer present in the upstream version